packages/ide-extension/src/panel/guidedAnswersPanel.ts

```diff
@@ -37,13 +37,15 @@ export class GuidedAnswersPanel {
      * @param [options] - optional options to initialize the panel
      * @param [options.ide] - optional runtime IDE (VSCODE/SBAS), default is VSCODE if not passed
      * @param [options.startOptions] - optional startup options like tree id or tree id + node id path
+     * @param [options.openToSide] - optional default is true, set to true to open the extension to the side
      */
     constructor(options?: Options) {
         this.startOptions = options?.startOptions;
         this.ide = options?.ide || 'VSCODE';
         const config = workspace.getConfiguration('sap.ux.guidedAnswer');
         const apiHost = config.get('apiHost') as string;
         const enhancements = getEnhancements(this.ide);
+        const openToSide = options?.openToSide || true;
```

Comment:
@theStiggySpd, are we sure the default should be set to `true`? 
That means if we start the command `SAP: Open Guided Answers` it will always show to the side, not full screen. I'd propose to have the default to `false` and if another extension calls the command it can decide to start to the side.

Reply:
I've discussed this with @theStiggySpd and got confirmation, we should set default value for `openToSide` to `false`

Reply:
Thanks for mentioning that. I'll change that to `false` :)